CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to 7 days to die sever manager
+You can find all the information about the server [here](https://docs.csmm.app/en/7D2D)
```

Comment:
Doesn't need a link to 7d2d servers here. This is a contributing guide so I think this will only confuse people.

7d2d is just the game server and not expressly needed for contributing. If someone unfamiliar with the project finds this line, they may think we are developing 7D2D but we are developing a server manager FOR 7d2d.